internal/pkg/utils/error_test.go

```diff
@@ -62,6 +62,59 @@ func TestMultiError(t *testing.T) {
 	assert.Equal(t, strings.TrimSpace(expected), e.Error())
 }
 
+func TestMultiError_CustomMsgFormat(t *testing.T) {
+	t.Parallel()
+	e := NewMultiError()
+	e.Append(fmt.Errorf(`12345`))
+
+	merged := NewMultiError()
+	merged.Append(fmt.Errorf("merged 1"))
+	merged.Append(fmt.Errorf("merged 2"))
+
+	sub := NewMultiError()
+	sub.Append(fmt.Errorf(`abc`))
+	sub.Append(fmt.Errorf(`def`))
+
+	sub1 := NewMultiError()
+	sub1.Append(fmt.Errorf("x"))
+	sub1.Append(fmt.Errorf("y"))
+	sub2 := NewMultiError()
+	sub2.Append(fmt.Errorf("z"))
+	sub3 := NewMultiError()
+	sub3.Append(fmt.Errorf("this is a very long line from error message, it is printed on new line"))
+
+	sub.AppendWithPrefix("sub1", sub1)
+	sub.AppendWithPrefix("sub2", sub2)
+	sub.AppendWithPrefix("sub3", sub3)
+
+	e.Append(merged)
+	e.AppendWithPrefix("my prefix", sub)
+	e.Append(fmt.Errorf("last error"))
+
+	// Custom function to modify message
+	f := NewErrorFormatter()
+	f.ErrorMessageFormatter(func(msg string) string {
+		return fmt.Sprintf("<<< %s >>>", msg)
+	})
+
+	expected := `
+- <<< 12345 >>>
+- <<< merged 1 >>>
+- <<< merged 2 >>>
+- <<< my prefix: >>>
+  - <<< abc >>>
+  - <<< def >>>
+  - <<< sub1: >>>
+    - <<< x >>>
+    - <<< y >>>
+  - <<< sub2: >>> <<< z >>>
+  - <<< sub3: >>>
+    - <<< this is a very long line from error message, it is printed on new line >>>
+- <<< last error >>>
+`
+	assert.Equal(t, strings.TrimSpace(expected), f.Format(e))
```

Comment:
Potrebujem aby API vracalo pekne chyby, po validacii `inputs`.
V Go je standardom, ze chyby zacinaju malym pismenom, a koncia bez bodky.

V API ich potrebujem naformatovat inak.
Preto som rozsiril kod ohladom formatovania chyb, aby sa dal lepsie nastavit.

V tomto teste pridavam ku kazdej chybe `<<< >>>`.

Defaultne spravanie sa nijak nezmenilo, je to pokryte ostatnymi/nezmenenymi testami v:
https://github.com/keboola/keboola-as-code/blob/9cd85ebada325445626af5aa577e79b311600048/internal/pkg/utils/error_test.go